dabuild.in

```diff
@@ -58,17 +58,16 @@ if [ ! \( -d "$ABUILD_PACKAGES" -a -w "$ABUILD_PACKAGES" \) ]; then
 fi
 
 ## setup volumes; use named volumes as cache if desired
-printf -v ABUILD_VOLUMES "%s "\
-       "-v ${HOME}/.abuild:/home/builder/.abuild" \
-       "-v ${PWD%/aports*}/aports:/home/builder/aports" \
-       "-v ${ABUILD_PACKAGES}:/home/builder/packages"
-
-if [ -s "/etc/alpine-release" ]; then
-  printf -v ABUILD_VOLUMES "%s "\
-         $ABUILD_VOLUMES \
-         "-v /etc/abuild.conf:/etc/abuild.conf"\
-         "-v /var/cache/distfiles:/var/cache/distfiles"\
-         "-v /var/cache/apk:/var/cache/apk"
+ABUILD_VOLUMES="-v ${HOME}/.abuild:/home/builder/.abuild \
+  -v ${PWD%/aports*}/aports:/home/builder/aports \
+  -v ${ABUILD_PACKAGES}:/home/builder/packages"
+
+if [ -f "/etc/abuild.conf" ]; then
+  ABUILD_VOLUMES="$ABUILD_VOLUMES -v /etc/abuild.conf:/etc/abuild.conf"
+fi
+
+if [ -w "/var/cache/distfiles" ]; then
+  ABUILD_VOLUMES="$ABUILD_VOLUMES -v /var/cache/distfiles:/var/cache/distfiles"
```

Comment:
We might want to store these in a volume in both cases.

Reply:
I think a bind mount would be nicer in this case.